server/src/common/model/schema/annuaire.js

```diff
@@ -1,26 +1,46 @@
 const annuaireSchema = {
   uai: {
     type: String,
-    default: null,
+    default: undefined,
     description: "L'UAI de l'établissement",
     unique: true,
   },
   siret: {
     type: String,
-    default: null,
+    default: undefined,
     description: "Le siret de l'établissement",
     unique: true,
   },
   nom: {
     type: String,
-    default: null,
+    default: undefined,
     description: "Le nom de l'établissement",
   },
   uais_secondaires: {
     type: Array,
     default: [],
     description: "La liste de tous les uais connus pour cet établissement",
   },
+  region: {
+    type: String,
+    default: undefined,
+    description: "Le code la région",
+  },
+  siegeSocial: {
+    type: Boolean,
+    default: undefined,
+    description: "Le siège social",
```

Comment:
C'est un booléen pour savoir si l'établissement est un siège social ?

Reply:
Ouai j'ai hésité et au final j'ai repris la réponse de l'API entreprise. Tu t'attendais à une string ?

Reply:
Ha oui, tu peux peut-être juste modifier la description ? 

```suggestion
    description: "Indique si l'établissement est le siège social",
```